core/db/migrate/20170323151450_add_missing_unique_indexes_for_unique_attributes.rb

```diff
@@ -32,7 +32,7 @@ def change
           end
 
           remove_index table_name, column if index_exists?(table_name, column)
-          add_index table_name, "lower(#{column})", unique: true
+          add_index table_name, column.downcase, unique: true
```

Comment:
@jafffmen it's `"lower(#{column})"` for values in table, not for column name. It's for unique lower case